spec/chainspecs/testnet/testnet.toml

```diff
@@ -0,0 +1,36 @@
+name = "ckb_testnet"
+
+[genesis]
+version = 0
+parent_hash = "0x0000000000000000000000000000000000000000000000000000000000000000"
+timestamp = 0
+txs_commit = "0x0000000000000000000000000000000000000000000000000000000000000000"
+txs_proposal = "0x0000000000000000000000000000000000000000000000000000000000000000"
+difficulty = "0x100"
+cellbase_id = "0x0000000000000000000000000000000000000000000000000000000000000000"
+uncles_hash = "0x0000000000000000000000000000000000000000000000000000000000000000"
+
+[genesis.seal]
+nonce = 0
+proof = [0]
+
+[params]
+initial_block_reward = 50000
+max_block_cycles = 100000000
+
+[pow]
+func = "Cuckoo"
+
+[pow.params]
+# the 2-log of the graph size, which is the size in bits of the node
+# identifiers
+edge_bits = 15
+
+# length of the cycle to be found, must be an even number, a minimum of 12 is
+# recommended
+cycle_length = 12
+
+# An array list paths to system cell files, which is absolute or relative to
+# the directory containing this config file.
+[[system_cells]]
+path = "cells/secp256k1_blake2b_lock"
```

Comment:
Does it support path like `../cells`. We may want to share the identical cells between specs.

Reply:
The crate we use doesn't support it, since internally what it does is just a key lookup process in a map. However, I believe we can [canonicalize](https://doc.rust-lang.org/std/path/struct.Path.html#method.canonicalize) in our code to solve this problem.

Reply:
The crate we use here doesn't support this, since all it does is a hashmap lookup. However, I think we can [clean](https://github.com/danreeves/path-clean) the path outside of the crate to support this behavior.